<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movieDesk</class>
 <widget class="QDialog" name="movieDesk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="mainCameraButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
rgb(19, 74, 255)</string>
   </property>
   <property name="text">
    <string>主</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viceButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(74, 116, 255);</string>
   </property>
   <property name="text">
    <string>副1</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>0</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:
url(:/image/image/return.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
